ci/travis/before_install-osx.sh

```diff
@@ -33,7 +33,15 @@ if [ $CC = 'clang' ]; then
     export CXX='clang++'
 fi
 
+echo "Installing Cap'n Proto..."
+curl -O https://capnproto.org/capnproto-c++-0.5.2.tar.gz
+tar zxf capnproto-c++-0.5.2.tar.gz
+cd capnproto-c++-0.5.2
+./configure
```

Comment:
special configure options had to be used, according to the readme. Is it not needed anymore? 
https://github.com/numenta/nupic.core/pull/579/files#diff-12994ec1ba63d2723056cdccf090c99eL14
https://github.com/numenta/nupic.core/pull/579/files#diff-12994ec1ba63d2723056cdccf090c99eL21